pytorch_forecasting/data/samplers.py

```diff
@@ -51,8 +51,9 @@ def __init__(
             )
         if not isinstance(drop_last, bool):
             raise ValueError(
-                "drop_last should be a boolean value, but got "
-                "drop_last={}".format(drop_last)
+                "drop_last should be a boolean value, but got drop_last={}".format(
```

Comment:
can we use `f"`?

Reply:
Yes, it would be better, but that could be done with adding a `UP` rule, see #1856